### PR TITLE
 TalkBack Accessibility Improvement Umbrella Tracker #7913

### PR DESCRIPTION
**Pull request by jindalshiva**
## Purpose / Description
_When the user enable TalkBack, then there were some issues with the UI of the app._

## Fixes
Fixes _https://github.com/ankidroid/Anki-Android/issues/7913_

## Approach
_I had to make changes in the XML files of the app, make some graphics and buttons of the app focusable_

## How Has This Been Tested?

Test this app by running this updated code, on your physical devices or emulator and also enable TalkBack on the device.

## Learning (optional, can help others)
_Started by learning about TalkBack. After this, I started making some changes to it._

_https://support.google.com/accessibility/android/answer/6283677?hl=en_

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by david-allison-1 — DISMISSED** [review]
I can't tap cards in the card browser after checking out this change

File: AnkiDroid/src/main/res/layout/my_account.xml [hunk 1]
```diff
@@ -78,7 +78,9 @@
                         android:layout_height="wrap_content"
                         android:inputType="textPassword"
                         android:focusable="true"
-                        android:layout_margin="@dimen/content_vertical_padding"/>
+                        android:layout_margin="@dimen/content_vertical_padding"
+                        android:paddingRight="40dp"/>
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/layout/my_account.xml:82]: Typically we want `paddingEnd` to ensure that right-to-left layouts handle the padding correctly.

Comment (jindalshiva) [reply AnkiDroid/src/main/res/layout/my_account.xml:82]: You want paddingEnd instead of paddingRight.

Comment (david-allison-1) [reply AnkiDroid/src/main/res/layout/my_account.xml:82]: If it makes sense in this context, then yes

Comment (jindalshiva) [reply AnkiDroid/src/main/res/layout/my_account.xml]: Hi @david-allison-1, just check, I have updated the code.

**Comment by jindalshiva** [comment]
> I can't tap cards in the card browser after checking out this change

Hi @david-allison-1, now you can tap on cards in the card browser.

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
This doesn't compile, CI is failing:

```
* What went wrong:
Execution failed for task ':AnkiDroid:mergeDebugResources'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > Android resource compilation failed
     /Users/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/layout/deck_item.xml:36: AAPT: error: not well-formed (invalid token).
         
     /Users/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/layout/deck_item.xml: AAPT: error: file failed to compile.
```

**Comment by mikehardy** [comment]
This PR is in a strange state from a GitHub perspective, I have no idea why the automated CI checks did not run 😅  - and it does not appear that I can cause them to run right now! Really strange. I would bet that adding some trivial change (like maybe a commit that added a comment, and then a second one that removed it - *just to force github to see it updated*) would cause the CI actions to run, otherwise I'm not sure what the best path forward is (open a new PR with the same commits perhaps? reviewer pull the branch locally and run `./gradlew clean jacocoTestReport lintRelease` ?). 

I don't see a recent incident https://www.githubstatus.com/history so I'm not sure what happened...

File: AnkiDroid/src/main/res/layout/card_item_browser.xml [hunk 2]
```diff
@@ -43,4 +43,4 @@
         android:paddingVertical="1dp"
         android:gravity="top"/>
 
-</LinearLayout>
\ No newline at end of file
+</LinearLayout>
```
Comment (vinishhub) [inline AnkiDroid/src/main/res/layout/card_item_browser.xml:46]: here it shows changes due to line ending differences.

File: AnkiDroid/src/main/res/layout/night_mode_switch.xml [hunk 3]
```diff
@@ -6,6 +6,7 @@
     <androidx.appcompat.widget.SwitchCompat
         android:id="@+id/switch_compat"
         android:layout_width="fill_parent"
+        android:focusable="true"
         android:contentDescription="@string/night_mode"
         android:layout_height="match_parent" />
 </LinearLayout>
```
Comment (vinishhub) [inline AnkiDroid/src/main/res/layout/night_mode_switch.xml:12]: here also the  line ending differences. you've must added a new line after this

**Pull request closed without merging**